plugins/@grouparoo/salesforce/src/lib/export/model.ts

```diff
@@ -32,8 +32,17 @@ export function getSalesforceModel(
 
   const modelKeys = Object.keys(model);
   const refKeys = modelKeys.filter((k) => k.indexOf("Reference") > 0);
```

Comment:
Might as well just make these `refKeys` explicit, too then (in an array of the 3)